pandas/tests/frame/test_analytics.py

```diff
@@ -466,6 +466,33 @@ def test_corrwith_mixed_dtypes(self):
         expected = pd.Series(data=corrs, index=['a', 'b'])
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.xfail
+    def test_corrwith_dup_cols(self):
+        # GH 21925
+        df1 = pd.DataFrame(np.vstack([np.arange(10)] * 3).T)
```

Comment:
can you add an example with an empty frame.

Reply:
A test to ensure that the output is an empty `Series` (with the proper index)?